lib/fathead/android_api_docs/parse.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin env python
+# -*- coding: utf-8 -*-
+from bs4 import BeautifulSoup
+from glob import glob
+import re
+
+def build_article(tr):
+    '''Builds fathead article entry.
+
+    '''
+
+    title = tr.td.a.contents[0]
+    url = (tr.find('a')['href'])
+    #Some packages don't have a description, but simply a page listing classes and interfaces (possibly enums)
+    try:
+        abstract = tr.find('p').contents[0]
+    except:
+        abstract = "More information about interfaces and classes at " + url
+    title = '<span class="prog__sub">%s</span>' % title
+    #Replace newline characters with a space
+    abstract = re.sub('\n', ' ', abstract)
```

Comment:
I actually didn't want the original newlines included in the API docs, that's why this is here. The final description in the AI will word-wrap, correct?